bindings/postgres/postgres.go

```diff
@@ -23,18 +23,24 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgxpool"
 
 	"github.com/dapr/components-contrib/bindings"
 	"github.com/dapr/components-contrib/metadata"
 	"github.com/dapr/kit/logger"
+
+	pgxuuid "github.com/jackc/pgx-gofrs-uuid"
+	pgxdecimal "github.com/jackc/pgx-shopspring-decimal"
 )
 
 // List of operations.
 const (
-	execOperation  bindings.OperationKind = "exec"
-	queryOperation bindings.OperationKind = "query"
-	closeOperation bindings.OperationKind = "close"
+	execOperation     bindings.OperationKind = "exec"
+	queryOperation    bindings.OperationKind = "query"
+	closeOperation    bindings.OperationKind = "close"
+	fetchOneOperation bindings.OperationKind = "fetchOne"
+	fetchAllOperation bindings.OperationKind = "fetchAll"
```

Comment:
What's the difference between fetchOne, fetchAll, and query?

Reply:
query: returns array of strings (with no column names)
fetchOne: returns map of key value pairs (with column names as key)
fetchAll: returns array of map of key value pairs (with column names as key)